pygmt/src/plot3d.py

```diff
@@ -36,7 +36,7 @@
     Y="yshift",
     Z="zvalue",
     a="aspatial",
-    i="columns",
+    i="incols",
```

Comment:
Since this is a backward-incompatible change, we need to use the `deprecate_parameter` decorator, similar to https://github.com/GenericMappingTools/pygmt/pull/1298.

Reply:
Added in b58698a